pinot-core/src/test/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManagerTest.java

```diff
@@ -133,6 +135,30 @@ private LLCRealtimeSegmentZKMetadata createZkMetadata() {
     return segmentZKMetadata;
   }
 
+  public static class FakeStreamConsumerFactory extends StreamConsumerFactory {
```

Comment:
Does it make sense to move this to another class/package so that we can use it else where?

Reply:
I didn't really find any other places which need it. I was planning to add fake stream if I encountered a lot of places that need the stream (with mock implementations)

Reply:
Added a FakeStream, to use in all the tests